src/main/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveService.java

```diff
@@ -64,4 +64,19 @@ public Oppgave opprettOppgave(Bruker bruker) {
 
         return oppgave;
     }
+
+    public Oppgave opprettOppgaveArbeidstillatelse(Bruker bruker) {
+
+        kontaktBrukerHenvendelseProducer.publiserHenvendelse(bruker.getAktorId());
+
+        String beskrivelse = String.format("");
```

Comment:
Forslag til tekst som Marthe har lagt inn i Trello-boardet: "Brukeren får ikke registrert seg som arbeidssøker pga. manglende oppholdstillatelse i Arena, og har selv opprettet denne oppgaven. Ring bruker og følg midlertidig rutine på navet om løsning for registreringen av arbeids- og oppholdstillatelse"